repo/packages/W/weavescope-probe/2/marathon.json.mustache

```diff
@@ -0,0 +1,57 @@
+{
+  "id": "{{service.name}}",
+  "cpus": {{weavescope.probe.cpus}},
+  "mem": {{weavescope.probe.mem}},
+  "instances": {{weavescope.probe.instances}},
+  "constraints": [
+    [
+      "hostname",
+      "UNIQUE"
+    ]
+  ],
+  "container": {
+    "type": "DOCKER",
+    "volumes": [
+      {
+        "containerPath": "/var/run/docker.sock",
+        "hostPath": "/var/run/docker.sock",
+        "mode": "RW"
+      }
+    ],
+    "docker": {
+      "image": "{{resource.assets.container.docker.48be3cfb15b2382f4055c867ccf477ed9a2b22ada18813c5f0d398753be7bf27}}",
```

Comment:
should be `resource.assets.container.docker.image` and the value defined in resource.json